src/Simulation/TargetDefinitions/TargetPackages/QsharpCore.Package.props

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I quite like how the used quantum instruction set becomes quite apparent this way.